conda_pack/core.py

```diff
@@ -386,6 +398,8 @@ def pack(
             # Ensure the prefix is a relative path
             arcroot = arcroot.strip(os.path.sep) if arcroot else ""
 
+        self._check_output_dir(output, force)
+
         if os.path.exists(output) and not force:
             raise CondaPackException("File %r already exists" % output)
```

Comment:
I could move this check inside the new method and rename it `_check_output_location()` or similar